teku/src/test/java/tech/pegasys/teku/cli/AbstractBeaconNodeCommandTest.java

```diff
@@ -41,8 +44,24 @@
 
   final StartAction startAction = mock(StartAction.class);
 
+  static class BeaconNodeCommandNoLogging extends BeaconNodeCommand {
+    public BeaconNodeCommandNoLogging(
+        PrintWriter outputWriter,
+        PrintWriter errorWriter,
+        Map<String, String> environment,
+        StartAction startAction) {
+      super(outputWriter, errorWriter, environment, startAction);
+    }
+
+    @Override
+    public void startLogging() {
+      // disable logging startup, so file appenders are not really created
+    }
+  }
```

Comment:
I wonder if instead of having to use a subclass of `BeaconNodeCommand` here we should just make the the `startLogging` method a non-static method on `LoggingConfigurator` (ie `startLogging(LoggingConfig config)` and then we can pass the instance of `LoggingConfigurator` into the `BeaconNodeCommand` constructor. For tests we'd pass a mock that does nothing.

Reply:
It could look cleaner, yeah